ServerRules.md

```diff
@@ -0,0 +1,11 @@
+## Server Rules
+
+1. Don't do anything you wouldn't do in Oso Negro
+2. Read and follow the Code of Conduct (https://nelson.social/about/more#code-of-conduct)
+3. No racist, sexist, homophobic, transphobic, ableist, or otherwise discriminatory or hateful jokes or language
+4. Don't post or threaten to post other people’s personally identifying information (i.e., doxing)
+5. Don't share intentionally false or misleading information
+6. Don't spam or engage in excessive commercial posting (business accounts are allowed, though)
```

Comment:
Keep it short, so people will read it

```suggestion
6. Don't spam or engage in excessive commercial posting
```

Reply:
I think we need to be clear about where we stand on this somewhere.

The current CoC says:

> A place for general advertisement. Promoting your original work is welcome, but please avoid overly commercial self-promotion

This is probably too vague. If I'm running a barbers shop or a bar in Nelson, can I make an account for my brand?

I've noticed that the historical "commercial stuff is bad" position of the mastodon community is [being challenged](https://hachyderm.io/@avdi/109541472311429413), and I think that's a good thing.

In our context, supporting our community, I think we absolutely need to give local businesses a platform to connect with their customers, I think that's part of our service and I think we need to say so *somewhere*.

But maybe we're not all on the same page about that. @blaine?